tiberius/src/query.rs

```diff
@@ -252,7 +256,7 @@ impl<I: BoxableIo> StmtResult<I> for ExecFuture<I> {
 
 /// A row in one resultset of a query
 #[derive(Debug)]
-pub struct QueryRow(TokenRow);
+pub struct QueryRow(pub TokenRow);
```

Comment:
This breaks BC and also I'd like this not exposed since it's protocol internals.
(e.g. in new TDS versions row might change)